istioctl/cmd/root.go

```diff
@@ -311,19 +310,20 @@ func configureLogging(_ *cobra.Command, _ []string) error {
 	return nil
 }
 
+// softGraduatedCmd is kept for future use when commands are softly graduated.
 // softGraduatedCmd is used for commands that have graduated, but we still want the old invocation to work.
-func softGraduatedCmd(cmd *cobra.Command) *cobra.Command {
-	msg := fmt.Sprintf("(%s has graduated. Use `istioctl %s`)", cmd.Name(), cmd.Name())
-
-	newCmd := *cmd
-	newCmd.Short = fmt.Sprintf("%s %s", cmd.Short, msg)
-	newCmd.RunE = func(c *cobra.Command, args []string) error {
-		fmt.Fprintln(cmd.ErrOrStderr(), msg)
-		return cmd.RunE(c, args)
-	}
-
-	return &newCmd
-}
+//func softGraduatedCmd(cmd *cobra.Command) *cobra.Command {
```

Comment:
should this be deleted vs commented out?

Reply:
I looked at the old codes, and they just removed this `softGraduatedCmd ` instead of commented out.

Reply:
Fixed, thanks